src/ankit/assignment_30/SecondBiggestNumber.java

```diff
@@ -0,0 +1,40 @@
+/*
+ Assignment - 30 : 4th May'2021
+
+WAP to find Second max from given array
+int[] arr = {10,33,43,55,97,11,3};
+ouput : 55 
+ */
+
+package ankit.assignment_30;
+
+import java.util.Arrays;
+
+public class SecondBiggestNumber {
+	void biggestNum(int[] intArr) {
+		int max = intArr[0];
+		for(int index = 0 ; index < intArr.length ; index++) {
+			for(int innerIndex = 1; innerIndex < intArr.length ; innerIndex++) {
+				if (max < intArr[innerIndex]) {
+					max = intArr[innerIndex];
+				}
+			}
+		}System.out.println(max);	
+	}
+
+	void secondBigNumber(int[] intArr) {
+		Arrays.sort(intArr); // array by default sorted in ascending order
```

Comment:
We will not use this direct function, we have to write logic

![Uploading Screenshot 2021-05-04 at 8.49.46 PM.png…]()